jenkins-pipelines/continueous-delivery-jenkinsfile

```diff
@@ -0,0 +1,62 @@
+pipeline {
+    agent any
+    environment {
+
+        DEV_USER = 'xxxxx'
+        DEV_HOST = 'xxxxxxx.com'
+        PRE_PROD_USER = 'xxxxx'
+        PRE_PROD_HOST = 'xxxxxx.com'
+        PROD_USER = 'xxxxx'
+        PROD_HOST = 'xxxxxxx.com'
+    }
+
+    stages {
+        stage('Deploy to Dev') {
+            steps {
+                copyArtifacts(projectName: 'testgrid-build');
```

Comment:
Please add a comment mentioning which files u actually get as part of this process.